modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java

```diff
@@ -390,79 +416,51 @@ private TrackSelectionStrategy getTrackSelectionStrategy(MediaPackage mediaPacka
     WorkflowOperationInstance operation = workflowInstance.getCurrentOperation();
     String strategyCfg = StringUtils.trimToEmpty(operation.getConfiguration(TRACK_SELECTION_STRATEGY)).toLowerCase();
 
-    TrackSelectionStrategy trackSelectionStrategy;
     if (strategyCfg.isEmpty()) {
-      trackSelectionStrategy = TrackSelectionStrategy.EVERYTHING; // "transcribe everything" is the default/fallback
-    } else {
-      try {
-        trackSelectionStrategy = TrackSelectionStrategy.fromString(strategyCfg);
-      } catch (IllegalArgumentException e) {
-        throw new WorkflowOperationException(String.format(
-            "Speech-to-Text job for media package '%s' failed, because of wrong workflow configuration. "
-                + "track-selection-strategy of type '%s' does not exist.", mediaPackage, strategyCfg));
-      }
+      return TrackSelectionStrategy.EVERYTHING; // "transcribe everything" is the default/fallback
```

Comment:
We can just return if we already know the result. No need to continue. This makes the structure simpler.